crates/eframe/src/native/file_storage.rs

```diff
@@ -41,16 +41,16 @@ impl Drop for FileStorage {
 
 impl FileStorage {
     /// Store the state in this .ron file.
-    fn from_ron_filepath(ron_filepath: impl Into<PathBuf>) -> Self {
+    pub fn from_ron_filepath(ron_filepath: impl Into<PathBuf>) -> Option<Self> {
```

Comment:
why return an Option for a function that never returns `None`?

```suggestion
    pub fn from_ron_filepath(ron_filepath: impl Into<PathBuf>) ->Self {
```

Reply:
Please see https://github.com/emilk/egui/pull/3939/commits/cdcfe7a263473eafabcb130b847bdcb281eef073#diff-99baf00dab62955842a4603f181399c5733a58a9a8201d8c7a09279b7870f0fdR49

It does return None here, through `?`.

Reply:
The motivation behind this change is, that I don't want people thinking that this will always return a valid `FileStorage`. (e.g. `read_ron` fails if it cannot access the file or cannot de-serialize the contents properly)